### ISSUE TITLE
False positive in D402

### ISSUE THREAD

**Issue by DeD1rk** [astral-sh/ruff#3769]
I stumbled upon a false positive for pydocstyle's D402 (First line should not be the function's signature) in ruff 0.0.259, python 3.8.10.

I'm not sure what pydocstyle itself would do, but it seems incorrect even if pydocstyle would also be triggered:

```python
class A(models.QuerySet):
    def b(self):
        """Call `C.b()` on something."""
        pass
```

Here the `b(`in the docstring seems to cause the violation. The rule is also triggered by `"""Call 'C.b(x)' on something."""`, `"""Call 'b(' on something."""` and `"""Call 'b(a,c)' on something."""`, but not if it does not contain `b(`.

**Comment by MichaReiser** [comment]
See https://github.com/astral-sh/ruff/pull/13388
